src/custom-types.json

```diff
@@ -837,6 +837,12 @@
   "text/x-scss": {
     "extensions": ["scss"]
   },
+  "text/x-python": {
+    "extensions": ["py", "pyx", "wsgi"],
+    "sources": [
+      "https://reposcope.com/mimetype/text/x-python"
```

Comment:
At best, this is not a primary source (rather, a secondary source), but I think it is actually the explicitly disallowed "type aggregating sites" as it seems I can change the URL to other mime types to see information about those as well (see https://reposcope.com/mimetype/text/plain). The link should be to how that site sourced their data on why those extensions are that mime type. The ideal primary source would be a link on the python project's own site.

Reply:
thanks for yr info, that mean i only can do in manually look up in my project I think